test/Test/EventListenerIntrospectionTraitTest.php

```diff
@@ -193,6 +194,13 @@ public function testAssertListenerAtPriorityFailsWhenListenerIsNotFound()
                     $arguments['event'],
                     $arguments['priority']
                 ), $e->getMessage(), sprintf('Assertion failure message was unexpected: %s', $e->getMessage()));
+            // for PHPUnit 5.7
```

Comment:
Please have a look how @weierophinney did it in another repo:
https://github.com/zendframework/zend-component-installer/commit/654983e8f01ddb636c1886fa1880db854d9cc0bd
I'd suggest the same way here.